Mathlib/Data/Finset/Image.lean

```diff
@@ -492,7 +492,6 @@ theorem image_symmDiff [DecidableEq α] {f : α → β} (s t : Finset α) (hf :
     (s ∆ t).image f = s.image f ∆ t.image f :=
   mod_cast Set.image_symmDiff hf s t
 
-@[simp]
```

Comment:
Why this change?

Reply:
It seems a bit of a strange simp lemma to me, since it introduces a metavariable `f` inside of the subgoal `h` that needs to be discharged. And this was slowing down the `simp` in the `aesop` at `mem_shadow_iterate_iff_exists_card`.